Makefile

```diff
@@ -1,15 +1,21 @@
 DESIGN_SYSTEM_VERSION=`cat .design-system-version`
 
-load-design-system-templates:
+load-design:
 	./scripts/load_release.sh onsdigital/design-system $(DESIGN_SYSTEM_VERSION)
 
-run: load-design-system-templates
-	flask --app application run
+run-ui:
+	poetry run flask --app application run --debug -p 8000
 
 format-python:
 	poetry run isort .
 	poetry run black .
-	poetry run flake8 .
+	poetry run flake8 --ignore=E501 .
```

Comment:
Instead of ignoring we could add a .flake8 file with line-length set the same as ruff/black (120?)

Reply:
+ .flake8